app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -46,7 +46,7 @@ def prov_field_changed
         if refresh_divs
           all_dialogs.each do |dialog_name, dialog|
             if dialog[:display] == :show && dialog_name == @edit[:new][:current_tab_key]
-              page.replace_html(dialog_name, :partial => dialog_partial_for_workflow, :locals => {:wf => @edit[:wf], :dialog => dialog_name})
+              page.replace_html(dialog_name, :partial => dialog_partial_for_workflow, :locals => {:wf => @edit[:wf], :dialog => dialog_name, :isDisabled => false})
```

Comment:
If we don't declare isDiasbled props, then default prop is false, right?
what is the need of declaring false here?

Reply:
`isDisabled` is default `true` see here https://github.com/ManageIQ/manageiq-ui-classic/pull/7879/files#diff-374bbbb7646dd6f8c1cc7ecb45e63c8f1a2890943627bdd87c9a1535cc810d76R25